docs/features/vision/fog_of_war_manager.md

```
@@ -0,0 +1,433 @@
+var _unit_to_circles_mapping : Dictionary = {}
+var _unit_to_shape_3d_mapping : Dictionary = {}
+
+@onready var _revealer : ColorRect = find_child("Revealer")
+@onready var _fog_viewport : SubViewport = find_child("FogViewport")
+@onready var _fog_viewport_container : SubViewportContainer = find_child("FogViewportContainer")
+@onready var _combined_viewport : SubViewport = find_child("CombinedViewport")
+@onready var _screen_overlay : MeshInstance3D = find_child("ScreenOverlay")
+@onready var _visibility_field : Area3D = find_child("VisibilityField")
+
+func _ready() -> void:
+	_screen_overlay.material_override.set_shader_parameter(
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)